src/style.css

```diff
@@ -0,0 +1,50 @@
+@font-face {
+  font-family: Avenir-Book;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+.wrapper {
+  margin-top: 20px;
+}
+
+input {
+  font-weight: 300;
+  width: 100%;
+  box-sizing: border-box;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  font-family: Avenir-Book, Helvetica, sans-serif;
+  font-size: 16px;
+  background-image: url("images/Search.svg");
+  background-position: 26px 25px;
+  background-repeat: no-repeat;
+  padding: 25px 62px;
+  height: 70px;
+  color: #3d4e61;
+}
+
+.search-input__small {
```

Comment:
```suggestion
.search-input--small {
```
Модификатор обычно записывается через `--`

Reply:
Кстати не вижу у тебя класса search-input, который ты тут модифицируешь

Reply:
![image](https://user-images.githubusercontent.com/38065883/140324807-3450f3e9-1edc-4c2b-ace1-454b67fdcef3.png)
Цвет текста в плейсхолдере все еще не правильный